### PR TITLE
Added flow support

### PR DESCRIPTION
**Pull request by kthorri**
#2 
The project is ready for flow with this pull request.

However I did not flowify the files, mostly because of missed information to run properly.

## Changes to workflow:
When running new code:
```
yarn build // This removes flow types from javascript when compiling
yarn start
```

To use IDE with Flow support you must enable usage of dependency Flow package

The executable code is now in "lib"

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "PiMonitor",
   "version": "1.0.0",
   "description": "Monitor for Raspberry Pi",
-  "main": "src/index.js",
+  "main": "lib/index.js",
```
Comment (kthorri) [inline package.json]: nice catch 🥇

**Comment by ciffelia** [comment]
Oops, I accidentally removed commits...
I'll re-push commits soon.

**Comment by ciffelia** [comment]
@kthorri Could you re-push your commits to kthorri:master so that I can re-push my commits? Sorry for your inconvenience:bow:

**Comment by kthorri** [comment]
Check out #4 :)

**Pull request closed without merging**